.github/workflows/CI.yml

```
@@ -0,0 +1,84 @@
+
+      - name: Configure Insecure Registry
+        run: |
+          sudo service docker stop
+          echo "{\"insecure-registries\": [\"${{ secrets.HARBOR_REG }}\"]}" | sudo tee /etc/docker/daemon.json
+          sudo service docker start
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+        with:
+          driver-opts: network=host
+          config-inline: |
```

actionlint (1.7.12)

[error] avoid using deprecated input "config-inline" in action "docker/setup-buildx-action@v3": Use buildkitd-config-inline instead

(action)